File: Networks/Project 3/Request/Request.java
public class Request{
   public int tml;
   public int requestID;
   public int operandCode;
   public int numberOperands;
   public int operand1;
   public int operand2;
   public int error;
   public int result;
   
   public Request(int tmlIn, int requestIDIn ,int operandCodeIn, int numberOperandsIn, int operand1In, int operand2In){
      tml = tmlIn;
      requestID = requestIDIn;
      operandCode = operandCodeIn;
      numberOperands = numberOperandsIn;
      operand1 = operand1In;
      operand2 = operand2In; 
   }
   
   public Request(int tmlIn, int requestIDIn ,int operandCodeIn, int numberOperandsIn, int operand1In){
      tml = tmlIn;
      requestID = requestIDIn;
      operandCode = operandCodeIn;
      numberOperands = numberOperandsIn;
      operand1 = operand1In; 
   }
   
   public Request(int tmlIn, int requestIDIn, int errorIn, int resultIn) {
      tml = tmlIn;
      requestID = requestIDIn;
      error = errorIn;
      result = resultIn;
   }
   public String toString() {
      final String EOLN = java.lang.System.getProperty("line.separator");
      String value = "";
      if(tml == 7){
         value = "TML = " + tml + " bytes" + EOLN +
                 "Request ID = " + requestID + EOLN +
                 "Error Code = " + error + EOLN +
                 "Result = " + result + EOLN;                 
               return value;
      }
      
      value = "TML = " + tml + " bytes" + EOLN +
                 "Request ID = " + requestID + EOLN +
                 "Operand Code = " + operandCode + EOLN +
                 "Number Operands = " + numberOperands + EOLN +
                 "Operand 1 = " + operand1 + EOLN;
      if(numberOperands == 2) {
         value += "Operand 2 = " + operand2 + EOLN;
      }
      
      return value;
   }

}

File: Networks/Project 2/Request/Request.cpp
#include <string>
#include <iostream>

using namespace std;
class Request{
   int tml;
   int requestID;
   int operandCode;
   int numberOperands;
   int operand1;
   int operand2;
   int error;
   int result;

   Request(int tmlIn, int requestIDIn ,int operandCodeIn, int numberOperandsIn, int operand1In, int operand2In){
      tml = tmlIn;
      requestID = requestIDIn;
      operandCode = operandCodeIn;
      numberOperands = numberOperandsIn;
      operand1 = operand1In;
      operand2 = operand2In;
   }

   Request(int tmlIn, int requestIDIn ,int operandCodeIn, int numberOperandsIn, int operand1In){
      tml = tmlIn;
      requestID = requestIDIn;
      operandCode = operandCodeIn;
      numberOperands = numberOperandsIn;
      operand1 = operand1In;
   }

   Request(int tmlIn, int requestIDIn, int errorIn, int resultIn) {
      tml = tmlIn;
      requestID = requestIDIn;
      error = errorIn;
      result = resultIn;
   }
   string toString() {
      string EOLN = "\n";
      string value = "";
      if(tml == 7){
         value = "TML = " + std::to_string(tml) + " bytes" + EOLN +
                 "Request ID = " + std::to_string(requestID) + EOLN +
                 "Error Code = " + std::to_string(error) + EOLN +
                 "Result = " + std::to_string(result) + EOLN;
               return value;
      }

      value = "TML = " + std::to_string(tml) + " bytes" + EOLN +
                "Request ID = " + std::to_string(requestID) + EOLN +
                 "Operand Code = " + std::to_string(operandCode) + EOLN +
                 "Number Operands = " + std::to_string(numberOperands) + EOLN +
                 "Operand 1 = " + std::to_string(operand1) + EOLN;
      if(numberOperands == 2) {
         value += "Operand 2 = " + std::to_string(operand2) + EOLN;
      }

      return value;
   }

};
